packages/dd-trace/test/appsec/remote_config/manager.spec.js

```diff
@@ -2,6 +2,7 @@
 
 const Capabilities = require('../../../src/appsec/remote_config/capabilities')
 const { UNACKNOWLEDGED, ACKNOWLEDGED, ERROR } = require('../../../src/appsec/remote_config/apply_states')
+const { expect } = require('chai')
```

Comment:
Maybe is just a personal opinion, so the change is optional, but I prefer when we require first external modules, and after our own modules.

```suggestion
const { expect } = require('chai')
const Capabilities = require('../../../src/appsec/remote_config/capabilities')
const { UNACKNOWLEDGED, ACKNOWLEDGED, ERROR } = require('../../../src/appsec/remote_config/apply_states')
```